client/end.py

```diff
@@ -0,0 +1,29 @@
+from definitions import *
+import pygame as pg
+
+from buttonPress import Button
+
+def endScreen(screen, address):
+    print("hello world")
+    button_rejoin = Button(90, 30, "Rejoin", (50,130))
+    backToLogin = Button(200, 30, "Return to Login Screen", (50,180))
+    
+    done = False
+    while not done:
+        for event in pg.event.get():
+            if event.type == pg.QUIT:
+                sys.exit(0)
```

Comment:
this is not an organic exit to the game client. pls follow the system used in gamesession (i think we have the same problem in login as well; pls fix all issues relating to this in its own commit)